cypress/integration/app-init-spec.js

```diff
@@ -0,0 +1,21 @@
+describe('app Initialization', () => {
+  Cypress.on('uncaught:exception', (err, runnable) => false);
+  it('Loads movieMockData on page load', () => {
```

Comment:
I messaged you how you can update this case to add some sense to it

Reply:
I tried one simple approach seeing documentation